SourceT/writehero_test.cpp

```diff
@@ -172,10 +174,10 @@ static void PackPlayerTest(PkPlayerStruct *pPack)
 	for (auto i = 0; i < 7; i++)
 		pPack->InvBody[i].idx = -1;
 	for (auto i = 0; i < MAXBELTITEMS; i++)
-		PackItemFullRejuv(pPack->SpdList+i, i);
+		PackItemFullRejuv(pPack->SpdList + i, i);
 	for (auto i = 1; i < 37; i++) {
-		if(spelldat_vanilla[i] != -1) {
-			pPack->pMemSpells |= (__int64)1 << (i-1);
+		if (spelldat_vanilla[i] != -1) {
+			pPack->pMemSpells |= (Uint64)1 << (i - 1);

```

Comment:
`1ULL` 

FAKE EDIT: Can't we just reuse the function for getting the bit of a spell, or even the macro?

Reply:
I think it's sometimes better to have the test logic disconnected from the application logic so you have less chance of reusing broken logic to very that it's broken.

Reply:
I actually didn't notice this was a test at first. I agree with your reasoning.